### ISSUE TITLE
Support for formatted json inside the table for helm configurations

### ISSUE THREAD

**Issue by raul1991** [rapidsai/frigate#17]
It would be nice to have the json formatted properly inside a table cell rather a single huge line of json.

**Comment by jacobtomlinson** [comment]
I agree this would be great. It is straight forward to update `json.dumps` to specify add indentation and sorting. Here's an [example from Stack Overflow](https://stackoverflow.com/questions/12943819/how-to-prettyprint-a-json-file). I would suggest an indentation of two would be appropriate here.

https://github.com/rapidsai/frigate/blob/d138d180c702165c1fd63b5a49ec90359d14282e/frigate/gen.py#L233

However this means that some will now run over multiple lines and so all the templates will need updating to support this.

https://github.com/rapidsai/frigate/blob/d138d180c702165c1fd63b5a49ec90359d14282e/frigate/templates/html.jinja2#L47
https://github.com/rapidsai/frigate/blob/d138d180c702165c1fd63b5a49ec90359d14282e/frigate/templates/markdown.jinja2#L22
https://github.com/rapidsai/frigate/blob/d138d180c702165c1fd63b5a49ec90359d14282e/frigate/templates/rst.jinja2#L30

**Comment by raul1991** [comment]
Just for my understanding ...

The {{default}} will be a multi line json so each block of it lying on separate line needs to back-ticked so as to form a correctly formatted json ?

**Comment by raul1991** [comment]
or probably using html code to create table like this 

https://stackoverflow.com/questions/28508141/code-block-inside-table-row-in-markdown/60502719#60502719

**Comment by raul1991** [comment]
Changed the code a bit and it looks beautiful with 4 indents, will push the code soon.

![image](https://user-images.githubusercontent.com/1281636/98447588-fc0e6e00-214b-11eb-87f6-4f5b99e32a9e.png)

**Comment by jacobtomlinson** [comment]
Awesome I look forward to reviewing the PR!
